.github/workflows/docker.yml

```diff
@@ -44,7 +51,9 @@ jobs:
           cache-to: type=inline
           platforms: linux/amd64,linux/arm64
           push: true
-          tags: geodynamics/aspect:latest
+          tags: |
+            ${{ github.repository_owner }}/${{ github.repository }}:latest
+            ghcr.io/${{ github.repository_owner }}/${{ github.repository }}:latest
```

Comment:
I am not sure if this matters, but does that mean that forks of aspect now automatically upload to the individual's ghcr?
What happens if dockerhub fails, will it still push to ghcr? Should ghcr maybe come first (more likely to be reliable than dockerhub)?